app/services/add_support_user.rb

```diff
@@ -0,0 +1,25 @@
+class AddSupportUser
```

Comment:
I don't feel strongly about this, but we are using form objects else where, that might be useful pattern to maintain, but again, I don't really mind.